config/consul_registr/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM consul
+
+COPY registr.json /consul/configs/
+
+ENTRYPOINT ["consul", "agent", "-config-dir=/consul/configs", "-data-dir=/tmp/consul"]
```

Comment:
Move this to infra/consul
It is not a part of positioning-service config

Reply:
Done